### ISSUE TITLE
[Feature] Horizontal/vertical lock

### ISSUE THREAD

**Issue by tharubberchicken** [redwarp/gif-wallpaper#248]
Could you perhaps implement the additional option to centre the GIF on just one axis? The double tap works great to centre both horizontal and vertical, but with larger GIFs I often find myself wanting to focus on a particular area while wanting to keep one axis precisely centred.

Perhaps a menu option, triple tap, or two finger tap. Just an idea, but I realise you would like to keep the app nice and simple, so I'm not sure how you'd  like to implement it.

Thanks for an awesome, beautiful app!

**Comment by redwarp** [comment]
Hum, maybe a two finger drag could be an option?

**Comment by tharubberchicken** [comment]
Yes, that could work well. 👍🏻 The app would be perfect then - at least for me!

**Comment by tharubberchicken** [comment]
Just wondering if you've thought any more about this feature. 'Cause I just spent 5 minutes trying to vertically align a landscape gif. 😁

**Comment by tharubberchicken** [comment]
![Screenshot_20230717_210120_Niagara Launcher](https://github.com/redwarp/gif-wallpaper/assets/91183771/5ab492ed-89ba-43e5-8eb8-4fc5896b3e2c)

**Comment by tharubberchicken** [comment]
![Screenshot_20230717_210054_Niagara Launcher](https://github.com/redwarp/gif-wallpaper/assets/91183771/890100ba-4a7c-4b32-9807-517b3971e544)

**Comment by tharubberchicken** [comment]
I deliberately unaligned it to illustrate. As you can see the second pic is nicely aligned to the top.👍🏻 A horizontal lock would enable me to shift the ninja to a preferred location without losing the top alignment.

**Comment by redwarp** [comment]
Sorry, I've been pretty bad at updating this app lately!
I should get back on the horse! (But Zelda is grabbing my time)
I can't give you an ETA, but thanks for the screenshots, I'll try to remotivate myself to do an update round

**Comment by tharubberchicken** [comment]
Zelda... I _totally_ get that. 😉 I'm glad I don't have a Switch, else I would be forever playing it too. We'll, I don't have a Switch, yet...
